### PR TITLE
AC-1066 Patient Details screen do not show long names properly

### PR DESCRIPTION
**Pull request by rishujam**
## Description of what I changed
Changed the constraints and width of text view of patient detail screen

## Issue I worked on
https://issues.openmrs.org/browse/AC-1066

## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishujam** [comment]
@shubhamsgit Please have a look

File: openmrs-client/src/main/res/layout/fragment_patient_details.xml [hunk 1]
```diff
@@ -57,18 +57,19 @@
...
-            android:layout_height="wrap_content"
-            android:layout_marginStart="20dp"
-            android:layout_marginLeft="20dp"
-            android:layout_marginTop="30dp"
-            android:fontFamily="@font/roboto_medium"
-            android:textColor="?android:attr/textColorPrimary"
-            android:textSize="18sp"
-            app:layout_constraintStart_toEndOf="@+id/patientPhotoCardView"
-            app:layout_constraintTop_toBottomOf="@id/deceasedView" />
+                android:id="@+id/patientDetailsName"
```
Comment (prateek27das) [inline openmrs-client/src/main/res/layout/fragment_patient_details.xml:60]: If the layout is part of a larger view hierarchy, consider using ConstraintLayout's optimization features.

**Review by prateek27das — COMMENTED** [review]
In general, it seems that the code is well-organized and has good readability.

**Comment by shubhamsgit** [comment]
> @shubhamsgit Please have a look

LGTM 👍

**Pull request merged**